psl/psl/tests/config/sources.rs

```diff
@@ -758,47 +739,6 @@ fn relation_mode_and_referential_integrity_can_coexist() {
     assert_eq!(config.relation_mode(), Some(RelationMode::ForeignKeys));
 }
 
-#[test]
-fn relation_mode_prisma_has_precedence_over_referential_integrity() {
-    let schema = indoc! {r#"
-        datasource ps {
-          provider = "sqlserver"
-          url = "mysql://root:prisma@localhost:3306/mydb"
-          relationMode = "prisma"
-          referentialIntegrity = "foreignKeys"
-        }
-
-        generator client {
-          provider = "prisma-client-js"
-          previewFeatures = ["referentialIntegrity", "relationMode"]
-        }
-    "#};
-
-    let config = parse_configuration(schema);
-
-    assert_eq!(config.relation_mode(), Some(RelationMode::Prisma));
-}
```

Comment:
This test is testing that the datasource properties can overlap. I think we should keep the test until we change this behaviour, which should be soon.

Reply:
Good catch